assets/src/infrastructure/ui/inputs/types.ts

```diff
@@ -20,7 +20,7 @@ export interface CommonInputProps<T = Element, V = React.ReactText> {
 	onChangeValue?: (value: V, event?: React.ChangeEvent<T> | React.FormEvent<T>) => void;
 }
 
-export interface ButtonProps extends Partial<ChakraButtonProps> {
+export interface ButtonProps extends Partial<ChakraButtonProps>, Pick<ChakraIconButtonProps, 'icon'> {
```

Comment:
getting test fails because of this (although locally my system says it's ok)
and my git is messed up right now with my transition to WSL and is taking me literally 5-10 minutes to perform any action, so I haven't been able to fix this yet